.github/workflows/contracts.yml

```diff
@@ -65,6 +68,14 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Setup Credentials
+        uses: webfactory/ssh-agent@v0.9.0
+        env:
+          SUPER_SECRET: ${{ secrets.deploy-keys }}
+        if: ${{ env.SUPER_SECRET != '' }}
+        with:
+          ssh-private-key: ${{ secrets.deploy-keys }}
+
```

Comment:
We'd like to avoid using an additional third party step, if possible (and only have it as a last resort).